frontend/src/components/App/PluginSettings/PluginSettings.tsx

```diff
@@ -1,12 +1,16 @@
-import { Switch } from '@mui/material';
+import { Switch, SwitchProps, Typography } from '@mui/material';
 import Box from '@mui/material/Box';
 import Button from '@mui/material/Button';
+import Link from '@mui/material/Link';
+import { styled } from '@mui/material/styles';
```

Comment:
We are trying to get rid of styles. Please use `@mui/system` or use the `sx` props.